docs/content/manual/_incoming/2288-2350-project-owner-should-not-see-alert copy.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Project owner should not see additional alert
+---
+
+* Related issues: [#2288](https://github.com/harvester/harvester/issues/2288) [BUG] The project-owner user will see an additional alert
+* * Related issues: [#2350](https://github.com/harvester/harvester/issues/2350) [Backport v1.0] The project-owner user will see an additional alert
```

Comment:
Should be only one `*`

Reply:
Thanks for the check, remove the duplicate `*`